ci/jenkins/jobs/projects.yaml

```diff
@@ -0,0 +1,43 @@
+- project:
+    name: antrea-cloud
+        jobs:
+              - '{name}-{test_name}-for-pull-request':
```

Comment:
What if I want to run the jenkins test nightly.. Wont it be suffixed with *-pull-request?